frontend/packages/component-library/src/accordion/__tests__/Accordion.test.tsx

```diff
@@ -0,0 +1,120 @@
+import {render, screen, fireEvent} from '@testing-library/react';
+import Accordion, {AccordionProps} from './../Accordion';
+
+const mockAccordionItems: AccordionProps['items'] = [
+  {
+    id: 'first',
+    label: 'First Accordion',
+    content: 'This is the first accordion content.',
+  },
+  {
+    id: 'second',
+    label: 'Second Accordion',
+    content: 'This is the second accordion content.',
+  },
+  {
+    id: 'third',
+    label: 'Third Accordion',
+    content: 'This is the third accordion content.',
+  },
+];
+
+describe('Accordion Component', () => {
+  it('renders all accordion items', () => {
+    render(<Accordion items={mockAccordionItems} />);
+
+    mockAccordionItems.forEach(({label}) => {
+      expect(screen.getByText(`${label}`)).toBeVisible();
+    });
+  });
+
+  it('should not show content initially', () => {
+    render(<Accordion items={mockAccordionItems} />);
+
+    mockAccordionItems.forEach(({content}) => {
+      expect(screen.queryByText(`${content}`)).not.toBeVisible();
+    });
+  });
+
+  it('should expand and collapse the accordion items', () => {
+    render(<Accordion items={mockAccordionItems} />);
+
+    const firstAccordion = screen.getByText('First Accordion');
+    expect(
+      screen.queryByText('This is the first accordion content.'),
+    ).not.toBeVisible();
+
+    fireEvent.click(firstAccordion);
+    expect(
+      screen.getByText('This is the first accordion content.'),
+    ).toBeVisible();
+
+    fireEvent.click(firstAccordion);
+    expect(
+      screen.queryByText('This is the first accordion content.'),
+    ).not.toBeVisible();
+  });
+
+  it('should allow multiple accordions to be opened separately', () => {
+    render(<Accordion items={mockAccordionItems} />);
+
+    const firstAccordion = screen.getByText('First Accordion');
+    const secondAccordion = screen.getByText('Second Accordion');
+
+    fireEvent.click(firstAccordion);
+    expect(
+      screen.getByText('This is the first accordion content.'),
+    ).toBeVisible();
+
+    fireEvent.click(secondAccordion);
+    expect(
+      screen.getByText('This is the second accordion content.'),
+    ).toBeVisible();
+  });
+
+  it('should close one accordion when another is opened if using <details>', () => {
```

Comment:
How is this test different than the one above it? Do we want other accordions to close if one is opened?

Reply:
Thanks for spotting! You're right, we don't need this test now. We've discussed with @moshe that we might want support this functionality as a prop, but that is for unknown future and right now we're not going to do this. Removing this test